packages/web-forms/e2e/README.md

```diff
@@ -0,0 +1,58 @@
+# E2E Tests for `@getodk/web-forms`
+
+This directory contains end-to-end (E2E) tests for the `@getodk/web-forms` package, which powers form filling and submission editing of ODK forms in a web browser. These tests use [Playwright](https://playwright.dev/) to simulate user interactions and ensure the package works as expected in real-world scenarios.
+
+## Folder Structure
+
+The E2E tests are located in `packages/web-forms/e2e/`. Here's the structure and purpose of each directory:
+
+```
+e2e/
+├── fixtures/         # Sample forms and data for testing.
+├── page-objects/     # Page Object Model structure for e2e tests, organizing UI abstractions into pages and reusable controls.
+    ├── controls/     # Reusable controls such as form fields, UI components, etc.
+        ├── GeopointComponent.ts  # Example of a reusable control for the geopoint question type.
```

Comment:
At the E2E level, I think it might be best to avoid "component" terminology here entirely. Since it's become a technical term of art, I worry that something like `GeopointComponent` might drift from association with a specific UI component.

Reply:
Sure! I changed it to control for these things: GeopointControl